app/src/types/user.js

```diff
@@ -42,13 +44,17 @@ export type User = t.TypeOf<typeof UserModel>;
  * @param from
  * @returns {User}
  */
-export function extractUserFromRequest(from: express$Request): User {
+export function extractUserFromRequest(from: express$Request): Option<User> {
   const reqWithUser = ((from: Object): { user: User });
```

Comment:
not sure whether you need this line anymore? I guess you can just say `from.user`?